contracts/src/precompiles/ArbOwner.sol

```diff
@@ -48,9 +48,15 @@ interface ArbOwner {
     /// @notice Get the network fee collector
     function getNetworkFeeAccount() external view returns (address);
 
+    /// @notice Get the infrastructure fee collector
+    function getInfraFeeAccount() external view returns (address);
```

Comment:
This should also be in `ArbOwnerPublic.sol`

Reply:
Oops, fixed.